spec/models/export/activity_iati_column_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe Export::ActivityIatiColumn do
+  describe "#headers" do
+    it "returns an array that contains the header name of the column" do
+      activities = []
+
+      column = described_class.new(activities: activities)
+
+      expect(column.headers).to eql ["Published on IATI"]
+    end
+  end
+
+  describe "#rows" do
+    context "when there are no activities" do
+      it "returns an empty hash" do
+        activities = []
+
+        column = described_class.new(activities: activities)
+
+        expect(column.rows).to be {}
+      end
+    end
+
+    context "when there are activities" do
+      context "when the activity has been published to IATI" do
```

Comment:
It would be more accurate to describe the spec as "when the activity is publishable to IATI" - that reflects the purpose of the field, and it might ease your unease about the past tense.

The past tense will still be present in the column header, but we've had confirmation that this is what works and makes sense for the client, so it's ok.